get-started/account.md

```diff
@@ -1,14 +1,19 @@
 ---
-layout: post
+layout: page
 title: How to Create and Edit Your User Account
 category: manual
 tags: help-manual my-account
 crumbname: Manage User Account
+date: June 23, 2020
+last_modified_at: June 23, 2020
 breadcrumbs:
```

Comment:
We should either update the Manual item in these breadcrumbs, or remove the breadcrumbs